cmd/delete/execution.go

```diff
@@ -0,0 +1,81 @@
+ | NAME (7)   | WORKFLOW NAME                                                           | TYPE     | PHASE     | STARTED                        | ELAPSED TIME  |
+  ------------ ------------------------------------------------------------------------- ---------- ----------- -------------------------------- --------------- 
+ | c6a51x2l9e | recipes.core.basic.lp.go_greet                                          | WORKFLOW | ABORTED   | 2021-02-17T08:13:04.680476300Z | 15.540361300s |
+  ------------ ------------------------------------------------------------------------- ---------- ----------- -------------------------------- --------------- 
+
+Terminate multiple executions with there names
+::
+
+ bin/flytectl delete execution eeam9s8sny p4wv4hwgc4  -d development  -p flytesnacks
+
+Similarly you can get executions to find the state of previously terminated executions.
+
+::
+
+ bin/flytectl get execution  -d development  -p flytesnacks
+  ------------ ------------------------------------------------------------------------- ---------- ----------- -------------------------------- --------------- 
+ | NAME (7)   | WORKFLOW NAME                                                           | TYPE     | PHASE     | STARTED                        | ELAPSED TIME  |
+  ------------ ------------------------------------------------------------------------- ---------- ----------- -------------------------------- --------------- 
+ | c6a51x2l9e | recipes.core.basic.lp.go_greet                                          | WORKFLOW | ABORTED   | 2021-02-17T08:13:04.680476300Z | 15.540361300s |
+  ------------ ------------------------------------------------------------------------- ---------- ----------- -------------------------------- --------------- 
+ | eeam9s8sny | recipes.core.basic.lp.go_greet                                          | WORKFLOW | ABORTED   | 2021-02-17T08:14:04.803084100Z | 42.306385500s |
+  ------------ ------------------------------------------------------------------------- ---------- ----------- -------------------------------- --------------- 
+ | p4wv4hwgc4 | recipes.core.basic.lp.go_greet                                          | WORKFLOW | ABORTED   | 2021-02-17T08:14:27.476307400Z | 19.727504400s |
+  ------------ ------------------------------------------------------------------------- ---------- ----------- -------------------------------- --------------- 
+
+Usage
+`
+)
+
+func terminateExecutionFunc(ctx context.Context, args []string, cmdCtx cmdCore.CommandContext) error {
+	for i := 0; i < len(args); i++ {
+		name := args[i]
+		logger.Infof(ctx, "Terminating execution of %v execution ", name)
+		_, err := cmdCtx.AdminClient().TerminateExecution(ctx, &admin.ExecutionTerminateRequest{
+			Id: &core.WorkflowExecutionIdentifier{
+				Project: config.GetConfig().Project,
+				Domain:  config.GetConfig().Domain,
+				Name:    name,
+			},
+		})
+		if err != nil {
```

Comment:
Should this also have continue on Error

Reply:
I thought about this option too 

In case of  register files where we have this continueOnerror option where user can 

cat1 ) give list of files from a folder using wildcard/archive and continue the registration on failures , this option seems very much required since without it the only way he has is to remove files in the folder /archive or have some neat way of filtering the bad files or wait until we have an option to accept the list of files from piped o/p

cat2) Another way  he can choose to pass the files is by explicitly passing each files absolute path . In this case , the option to remove is much easier since we cant expect him to pass in a long list of files manually .

The execution failures as of now are similar to cat2) where he specifies the list of execution id's and hence removing the failed and  reattempting them is easier.

But definitely it would be good to have this option and will take it up in next MR.